### ISSUE TITLE
[Accessibility] Search results, facets not operable with keyboard, WCAG 2.0 Success Criteria 2.1.1

### ISSUE THREAD

**Issue by spudthebud** [DOAJ/doaj#2169]
On a search results page (for a journal), it is not possible to use the keyboard to tab to the facets.

###Reproduce 
1. Do a journal search
2. Press the tab key to try to get to the first checkbox in the facets on the search results page

I was not able to do it.

WCAG 2.0 [Success Criteria 2.1.1 ](https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html) reminds us that content should be operable using the keyboard (not everyone can use a mouse)

OS: Windows 11
Browser: Chrome 106

**Comment by dommitchell** [comment]
Thank you for reporting this. We have duplicated the bug to our bug repo and I will update here once your ticket has been considered.

**Comment by dommitchell** [comment]
@spudthebud this one went live on our site yesterday.
